orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/WaitForUpInstancesTask.groovy

```diff
@@ -219,4 +240,24 @@ class WaitForUpInstancesTask extends AbstractWaitingForInstancesTask {
 
     return deployment?.capacity as Map<String, Integer>
   }
+
+  public static class Splainer {
+    List<String> messages = new ArrayList<>()
+
+    def add(String message) {
```

Comment:
a thought would be making it easy to do logstash StructuredArguments stuff in this message signature. As an example from above being able to tokenize out the serverGroupName seems like it could be nice for log searching later